transport/tchannel/handler.go

```diff
@@ -49,12 +49,9 @@ type inboundCall interface {
 	ShardKey() string
 	RoutingKey() string
 	RoutingDelegate() string
-
```

Comment:
Why all this blank lines are removed? Let's leave them, please. (Throughout the whole PR)